smoke-test/src/test/java/org/opennms/smoketest/graph/GraphSearchRestServiceIT.java

```diff
@@ -0,0 +1,161 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2019-2019 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2019 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.smoketest.graph;
+
+import static com.jayway.awaitility.Awaitility.await;
+import static io.restassured.RestAssured.given;
+import static io.restassured.RestAssured.preemptive;
+
+import java.util.concurrent.TimeUnit;
+
+import org.hamcrest.Matchers;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.opennms.smoketest.OpenNMSSeleniumTestCase;
+import org.opennms.smoketest.topo.GraphMLTopologyIT;
+import org.opennms.smoketest.utils.KarafShell;
+import org.opennms.smoketest.utils.RestClient;
+import org.opennms.test.system.api.NewTestEnvironment;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+
+public class GraphSearchRestServiceIT  extends OpenNMSSeleniumTestCase {
+
+	private final static Logger LOG = LoggerFactory.getLogger(GraphSearchRestServiceIT.class);
```

Comment:
Formatting issue.

Reply:
what is the issue?

Reply:
<img width="711" alt="Screenshot 2019-06-19 at 18 03 06" src="https://user-images.githubusercontent.com/4202259/59781652-87b7fe00-92bc-11e9-840f-394eb39606df.png">

The spacing is off.

Reply:
fixed.